x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.ts

```diff
@@ -63,3 +65,23 @@ export const getAgentPolicySummaryHandler = function (
     });
   };
 };
+
+export const getPolicyListHandler = function (
+  endpointAppContext: EndpointAppContext
+): RequestHandler<undefined, GetPackagePoliciesRequest['query'], undefined> {
+  return async (context, request, response) => {
+    const soClient = context.core.savedObjects.client;
+    const packagePolicyService = endpointAppContext.service.getPackagePolicyService();
+    try {
+      const listResponse = await packagePolicyService.list(soClient, request.query);
+
+      return response.ok({
+        body: listResponse,
+      });
+    } catch (error) {
+      return response.notFound({
```

Comment:
I am not yet familiar with the way we do server API so let me know is this is the way we do things:

You are assuming here the reason why the request failed is because it was not found (404) but the error can be that and anything else. I'd argue that a 500 error makes more sense here since the error was unexpected and the API itself does exist.

Reply:
agreed - this should be a `500` here.